rcll-refbox.spec

```diff
@@ -56,17 +56,18 @@ make %{?_smp_mflags} \
 mkdir -p %{buildroot}/%{_bindir}
 mkdir -p %{buildroot}/%{_libdir}
 mkdir -p %{buildroot}/%{_datadir}/rcll-refbox/{games,msgs}
-mkdir -p %{buildroot}/%{_sysconfdir}/rcll-refbox
+mkdir -p %{buildroot}/%{_sysconfdir}/rcll-refbox/cfg
 
-install -p ./bin/* %{buildroot}/%{_bindir}/
+find ./bin \( -name "llsf*" -o -name "rcll*" \) -exec install -p '{}' %{buildroot}/%{_bindir}/ \;
 find ./lib -type f -exec install -p '{}' %{buildroot}/%{_libdir}/ \;
 cp -a ./src/games/* %{buildroot}/%{_datadir}/rcll-refbox/games
 install -p ./src/msgs/*.proto %{buildroot}/%{_datadir}/rcll-refbox/msgs
-# TODO: This should be installed in RESDIR nad should be handled by make install.
+# TODO: This should be installed in RESDIR and should be handled by make install.
 mkdir -p %{buildroot}/%{_datadir}/rcll-refbox/libs/websocket/message_schemas
 install -p ./src/libs/websocket/message_schemas/*.json %{buildroot}/%{_datadir}/rcll-refbox/libs/websocket/message_schemas
-install -p ./cfg/* %{buildroot}/%{_sysconfdir}/rcll-refbox
-
+cd cfg
+find * -type f -exec install -Dp "{}" "%{buildroot}/%{_sysconfdir}/rcll-refbox/{}" \;
```

Comment:
Similarly here, why use `find` to install the files?

Reply:
because when i tried it with install, the subdirectory structure was not preserved, so i oriented myself using https://unix.stackexchange.com/a/165471.

Reply:
I see, then why not just use `cp -a` instead?

Reply:
Yes `install` is just `cp` with some options preset, see https://stackoverflow.com/a/39295609/2785041 for some more details :)